lessons/Early-Math/associative-property/langs/en-us/index.adoc

```diff
@@ -0,0 +1,123 @@
+= The Associative Property
+
+@description{This lesson introduces students to the Associative Property by way of Circles of Evaluation.}
+
+@lesson-prereqs{commutative-property}
+
+@keywords{Associative Property}
+
+[@lesson-intro-table]
+|===
+
+| Lesson Goals
+| Students will be able to...
+
+* Recognize that, no matter how we group numbers being either added or multiplied, the end result will always be the same.
+* Use the Associative Property to simplify expressions and make computation easier.
+
+| Student-facing Lesson Goals
+|
+
+* Let's explore the Associative Property via Circles of Evaluation.
+
+| Materials
+|[.materials-links]
+@material-links
+
+| Key Points For The Facilitator
+|
+* The Associative Property - like other laws of arithmetic - is more meaningful and memorable when students view it as a powerful tool to help them to do math efficiently.
+* Students may need support in parsing expressions that include both an operation that _does_ support Associativity along side an operation where Associativity does _not_ apply.
+|===
+
+== The Associative Property @duration{30 minutes}
+
+=== Overview
+
+Students explore Circles of Evaluation to discover that no matter how we group numbers being either added or multiplied, the end result will always be the same.
+
+=== Launch
+
+When asked to multiply @math{2 \times 17 \times 5}, the student who notices that @math{2 \times 5 = 10} will arrive at a result much more efficiently than a student who does not. This is the power of the @vocab{Associative Property}!
+
+In the short term, familiarity with the Associative Property enables students to make mental computations more efficiently. In the long term, comfort with this law of arithmetic (and others!) will provide a strong foundation for more complex algebraic reasoning.
+
+A note: We view the Associative Property as the @vocab{Commutative Property} *2.0*! With the Associative Property, students can not only change the order of the values in a multiplication or addition expression... they can change the _groupings_. _Finding the best groupings, students will discover, sometimes requires changing the order as well._
```

Comment:
recommmed removing the second sentence here. Feel like it conflicts with the third.